### PR TITLE
Downgrade unity

### PR DESCRIPTION
**Pull request by Ronserruya**
Need to downgrade unity to push a version that will run on older versions as well.

+ Updated docs
+ Updated sample app to check if an account already exists

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,9 @@ allprojects {
 dependencies {
 
     ...
+        implementation 'com.github.kinecosystem.kin-sdk-android:kin-sdk-lib:1.0.5'
```
Comment (yosriz) [inline README.md:29]: is it before or after b&r changes (why versions were not updated already?)

**Review by yosriz — APPROVED** [review]
what's this docs PDF file?

File: README.md (hunk 1 above)
Comment (Ronserruya) [reply README.md:29]: Its after, I just forgot to update the docs

**Comment by Ronserruya** [comment]
@yosriz Its exactly like the readme file, but we also include a pdf of it in the asset

**Pull request merged**